gpbft/gpbft.go

```diff
@@ -544,6 +548,17 @@ func (i *instance) tryConverge() error {
 		return nil
 	}
 
+	// Find the first initial proposal prefix that has strong quorum as a result of
+	// arriving QUALITY messages and update candidates for its consecutive prefixes.
```

Comment:
Will this select the smallest or the largest prefix? Which one would be the correct one in either case?

Reply:
It should be the longest prefix with strong quorum IIUC. I have refactored this locally which I'll push shortly.